src/transport/ipc-transport.js

```diff
@@ -53,7 +53,7 @@ IpcTransport.prototype.submitRpcRequest = function (rpcJso, errorCallback) {
     });
   } catch (error) {
     if (error.code === "EPIPE") error.retryable = true;
-    errorCallback(error);
+    setTimeout(function() { errorCallback(error); });
```

Comment:
Interesting, that makes sense.  When I've used `setTimeout` to force something to execute asynchronously, I always use `setTimeout(() => {/* hilarity ensues */}, 0)`.  I guess it accomplishes the same thing if you just leave out the 2nd parameter though?